src/types.jl

```diff
@@ -73,14 +73,18 @@ mutable struct Acb <: Number
     end
 end
 
+cprefix(::Type{Mag}) = :mag
+cprefix(::Type{Arf}) = :arf
+cprefix(::Type{Arb}) = :arb
+cprefix(::Type{Acb}) = :acb
+
 for (T, prefix) in ((Mag, :mag), (Arf, :arf), (Arb, :arb), (Acb, :acb))
     arbstruct = Symbol(prefix, :_struct)
     spref = "$prefix"
     @eval begin
         cstructtype(::Type{$T}) = $arbstruct
     end
     @eval begin
-        cprefix(::Type{$T}) = Symbol($spref)
```

Comment:
yeah I was lazy and quoting symbols is nasty: `cprefix(::Type{$T}) = $(QuoteNode(Symbol(prefix))))` gives
```julia
julia> @code_lowered cprefix(Arb)
CodeInfo(
1 ─     return :arb
)
```

Reply:
This looks nasty indeed, but I'll change to it!